docs/orchestrating-elastic-stack-applications/enterprise-search.asciidoc

```diff
@@ -0,0 +1,242 @@
+apiVersion: enterprisesearch.k8s.elastic.co/v1beta1
+kind: EnterpriseSearch
+metadata:
+  name: enterprise-search-quickstart
+spec:
+  version: {version}
+  count: 1
+  elasticsearchRef:
+    name: quickstart
+EOF
+----
+
+. Monitor the Enterprise Search deployment.
++
+Retrieve details about the Enterprise Search deployment:
++
+[source,sh]
+----
+kubectl get enterprisesearch
+----
++
+[source,sh,subs="attributes"]
+----
+NAME                            HEALTH    NODES    VERSION   AGE
+enterprise-search-quickstart    green     1        {version}      8m
+----
++
+List all the Pods belonging to a given deployment:
++
+[source,sh]
+----
+kubectl get pods --selector='enterprisesearch.k8s.elastic.co/name=enterprise-search-quickstart'
+----
++
+[source,sh]
+----
+NAME                                                READY   STATUS    RESTARTS   AGE
+enterprise-search-quickstart-ent-58b84db85-dl7c6    1/1     Running   0          2m50s
+----
++
+. Access logs for that Pod:
++
+[source,sh]
+----
+kubectl logs -f enterprise-search-quickstart-ent-58b84db85-dl7c6
+----
+
+. Access Enterprise Search
++
+A ClusterIP service is automatically created for the deployment,
+and can be used to access the Enterprise Search API from within the Kubernetes cluster:
++
+[source,sh]
+----
+kubectl get service enterprise-search-quickstart-ent-http
+----
++
+Use `kubectl-portforward` to access Enterprise Search from your local workstation:
++
+[source,sh]
+----
+kubectl port-forward service/enterprise-search-quickstart-ent-http 3002
+----
++
+Open `https://localhost:3002` in your browser.
++
+NOTE: Your browser will show a warning because the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser. Acknowledge the warning for the purposes of this quickstart but it is highly recommended that you <<{p}-enterprise-search-expose,configure valid certificates>> for any production deployments.
++
+Login as the `elastic` user created link:../k8s-quickstart.html[with the Elasticsearch cluster]. Its password can be obtained with:
++
+[source,sh]
+----
+kubectl get secret quickstart-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode; echo
+----
+
+[id="{p}-enterprise-search-configuration"]
+== Configuration
+
+[id="{p}-enterprise-search-upgrade-specification"]
+=== Upgrade the Enterprise Search specification
+
+Any setting can be changed in the Enterprise Search yaml specification, including version upgrades.
+
+ECK detects those changes and ensures a smooth rolling upgrade of all Enterprise Search Pods.
+
+[id="{p}-enterprise-search-custom-configuration"]
+=== Customize Enterprise Search configuration
+
+ECK sets up a default Enterprise Search configuration automatically.
+It can can be customized with link:https://www.elastic.co/guide/en/enterprise-search/current/configuration.html#configuration[Enterprise Search configuration] through the `config` element in the specification.
```

Comment:
Duplicate "can", but the sentence has a bigger issue. I recommend moving the link to Enterprise Search to the sentence above, so they read like this:

> ECK sets up a default [Enterprise Search configuration](https://www.elastic.co/guide/en/enterprise-search/current/configuration.html#configuration). Customize it through the `config` element in the specification.